docs/conf.py

```diff
@@ -248,3 +248,11 @@
 # -- Substutition variables
 
 rst_epilog = f".. |stac_version| replace:: {STACVersion.DEFAULT_STAC_VERSION}"
+
+nitpick_ignore = [
+    ("py:class", "Datetime"),
+    ("py:class", "L"),
+    ("py:class", "pystac.summaries.T"),
+    ("py:class", "HREF"),  # this one partially works
```

Comment:
this was killing me. It works in some cases but not others. It was incredibly frustrating to try to figure out and in the end I gave up.